nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
+
+## Graphical Installation {#sec-installation-graphical}
+
+The graphical installer is recommended for desktop users and will guide you
+through the installation.
+
+1.   In the "Welcome" screen, you can select the language of the Installer and
+     the installed system.
+
+     ::: {.tip}
+     Leaving the language as "American English" will help to be able to copy
+     and paste error messages in a search engine or issue report.
+     :::
+
+2.   Next you should choose your location to have the timezone set correctly.
+     You can actually click on the map!
+
+     ::: {.note}
+     The installer will use an online service to guess your location based on
+     your public IP address.
+     :::
+
+3.   Then you can select the keyboard layout. The default keyboard model should
+     work well with most desktop keyboards. If you have a special keyboard or
+     notebook, your model might be in the list. Select the language you are most
+     comfortable typing in.
+
+     ::: {.tip}
+     There are special keyboard layouts that are optimized to be ergonomic like
+     [NEO](https://www.neo-layout.org/),
+     [Dvorak](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout) and
+     [Colemak](https://colemak.com/), in opposition to the classic QWERTY layout
+     which comes from typewriters and has not changed much since 1873.
+     You might want to check them out.
+     :::
+
+4.   On the "Users" screen, you have to type in your name, login name and
+     password. You can also enable an option to automatically login to the
+     desktop.
+
+5.   Then you have the option to choose a desktop environment. If you want to
+     create a custom setup with a window manager, you can select "No desktop".
+
+     ::: {.tip}
+     If you don't have a favorite desktop and don't know which one to choose,
+     you can stick to either GNOME or Plasma. They have a quite different
+     design, so you should be able to choose which one you like better.
+     They are both popular choices and well tested on NixOS.
+     :::
+
+6.   You have the option to allow unfree software in the next screen.
+
+7.   The easiest option in the "Partitioning" screen is "Erase disk", which will
+     delete all data from the selected disk and install the system on it.
+     Also select "Swap (with Hibernation)" in the dropdown below it.
+     You have the option to encrypt the whole disk with LUKS.
+
+     ::: {.note}
+     At the top left you see if the Installer was booted with BIOS or UEFI. If
+     you know your system supports UEFI and it shows "BIOS", reboot with the
+     correct option.
+     :::
+
+     ::: {.warning}
+     Make sure you have selected the correct disk at the top and that no valuable
+     data is still on the disk!
+     :::
+
+8.   Check the choices you made in the "Summary" and click "Install".
+
+     ::: {.note}
+     The install takes about 15 minutes. The time varies based on the selected
+     desktop, internet speed and disk write speed.
```

Comment:
```suggestion
     The installation process can take about 15 minutes. The time varies based on the selected
     desktop environment, internet connection speed and disk read/write speed.
```

Reply:
it can take 8 minutes, it can take 10 minutes and it can take also 14 minutes, from my tests. but i have high-end hardware (nvme SSD) and gigabit internet, so i have choosen a higher number. if it's even faster, the user will be happy, but when it takes longer than this estimate, they might be unhappy. it says "takes **about** 15 minutes", so it's clear that it is not an exact value

i don't mentioned USB flash drive read speed and processor / memory speed, as it is not that relevant (install is mostly from internet and not much from usb drive i think). the section does not have to get so long